src/languages/singlestoredb/singlestoredb.keywords.ts

```diff
@@ -0,0 +1,852 @@
+import { flatKeywordList } from '../../utils';
+
+export const keywords = flatKeywordList({
+  // https://docs.singlestore.com/managed-service/en/reference/sql-reference.html
+  // https://docs.singlestore.com/managed-service/en/reference/sql-reference/restricted-keywords/list-of-restricted-keywords.html
```

Comment:
I wonder over the meaning of this "restricted keywords" list in SingleStoreDB documentation.

Are all these keywords actually supported by SingleStoreDB in the sense that they serve some function, or are they just reserved, in the sense that one can't use them as names of identifiers? The docs of SingleStoreDB aren't really clear on this matter:

> Keywords that are restricted (or reserved) require special treatment if they need to be used as an identifier such as a table or column name. This is because their specific meaning **could** cause issues when used in the context of such a name.

Reply:
I would say that all of them they are acting as reserved keywords. A subset of those is used as functions. (I have listed a couple of links in another discussion about which functions are documented).